process_cohort.local.sh

```diff
@@ -5,14 +5,18 @@ umask 002
 
 COHORT=$1
 mkdir -p cohorts/${COHORT}/
-LOCKFILE=cohorts/${COHORT}/process_cohort.lock
+# LOCKFILE=cohorts/${COHORT}/process_cohort.lock
 
 # add lockfile to directory to prevent multiple simultaneous jobs
-lockfile -r 0 ${LOCKFILE} || exit 1
-trap "rm -f ${LOCKFILE}; exit" SIGINT SIGTERM ERR EXIT
+# lockfile -r 0 ${LOCKFILE} || exit 1
+# trap "rm -f ${LOCKFILE}; exit" SIGINT SIGTERM ERR EXIT
```

Comment:
Shouldn't be necessary.  `lockfile` is a dependency installed by conda in the base snakemake conda environment, according to the tutorial instructions.

If we remove lockfile, we should let snakemake handle the lockfile handling by remove `--nlock` from the snakemake call.

Reply:
letting snakemake handle lockfiles